src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -111,4 +112,12 @@ export class SortableTableComponent implements OnInit {
     this.columnToSortBy = this.columns[indexOfColumnToSort].header;
     this.sortRows();
   }
+
+  getFont(cellData: SortableTableCellData): String {
+    if (cellData.font === undefined) {
+      return '';
+    }
+
+    return cellData.font;
+  }
```

Comment:
Can ask if this UI will expect different fonts from different data set?

Reply:
Only timestamps will be using a different font. The `cellData .font` for other data sets will be undefined and they will use the font in `styles.scss`